pkgs/by-name/nu/nu_scripts/package.nix

```
@@ -7,13 +7,13 @@
 
 stdenvNoCC.mkDerivation rec {
   pname = "nu_scripts";
-  version = "0-unstable-2025-04-14";
+  version = "0-unstable-2025-04-23";
 
   src = fetchFromGitHub {
     owner = "nushell";
     repo = pname;
```

Literal `repo` name is generally preferred.

```suggestion
    repo = "nu_scripts";
```

---

Done